src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -8336,6 +8336,11 @@ class StubGenerator: public StubCodeGenerator {
       UnsafeCopyMemory::create_table(8);
     }
 
+    // Initialize table for fill memory check.
+    if (UnsafeCopyMemory::_table == nullptr) {
+      UnsafeCopyMemory::create_table(8);
+    }
+
```

Comment:
Did you mean to initialize UnsafeSetMemory::_table here instead?

Reply:
Yes.  Good catch.